frontend/utilities/helpers.ts

```diff
@@ -50,14 +50,26 @@ const ADMIN_ATTRS = ["email", "name", "password", "password_confirmation"];
 
 export const addGravatarUrlToResource = (resource: any): any => {
   const { email } = resource;
+  const gravatarAvailable =
+    localStorage.getItem("gravatar_available") === "true";
 
   const emailHash = md5(email.toLowerCase());
-  const gravatar_url = `https://www.gravatar.com/avatar/${emailHash}?d=${encodeURIComponent(
-    DEFAULT_GRAVATAR_LINK
-  )}&size=200`;
-  const gravatar_url_dark = `https://www.gravatar.com/avatar/${emailHash}?d=${encodeURIComponent(
-    DEFAULT_GRAVATAR_LINK_DARK
-  )}&size=200`;
+
+  let gravatar_url;
+  let gravatar_url_dark;
+
+  if (gravatarAvailable) {
+    gravatar_url = `https://www.gravatar.com/avatar/${emailHash}?d=${encodeURIComponent(
+      DEFAULT_GRAVATAR_LINK
+    )}&size=200`;
+    gravatar_url_dark = `https://www.gravatar.com/avatar/${emailHash}?d=${encodeURIComponent(
+      DEFAULT_GRAVATAR_LINK_DARK
+    )}&size=200`;
+  } else {
+    gravatar_url = DEFAULT_GRAVATAR_LINK;
+    gravatar_url_dark = DEFAULT_GRAVATAR_LINK_DARK;
+  }
+
```

Comment:
Isn't the default link (which points to fleetdm.com) still outside the firewall?

Reply:
Unfortunately not. We pass our URL to gravatar, who then pulls it down and re-serves it from a gravatar.com URL, which gets blocked :(

Reply:
And so the fallback is to use the fleetdm link directly, but are we sure that won't also be blocked?

Reply:
Good catch, this should be a relative link not a reference to fleetdm.com. Will fix.

Reply:
@gillespi314 Fixed!